cirq-core/cirq/contrib/svg/svg.py

```diff
@@ -23,10 +23,7 @@ def fixup_text(text: str):
     if '[cirq.VirtualTag()]' in text:
         # https://github.com/quantumlib/Cirq/issues/2905
         return text.replace('[cirq.VirtualTag()]', '')
```

Comment:
Should we change the if statements above as well so that we don't return as soon as a single `if` matches?

Reply:
Sure, of course!  Done.  I have left the first short-circuit if as it does not matter when the string is dropped.